app/src/main/res/values/userpreferences.xml

```diff
@@ -16,6 +16,11 @@
   <string name="userprefs_submission_thumbnails">Submission thumbnails</string>
   <string name="userprefs_submission_thumbnail_summary_on">Show if available</string>
   <string name="userprefs_submission_thumbnail_summary_off">Always hidden</string>
+
+  <string name="userprefs_item_byline_comment_count">Submission comments count</string>
+  <string name="userprefs_item_byline_comment_count_summary_on">Show comments count</string>
```

Comment:
Looking at other settings I see that they mostly avoid duplicating text from titles. Perhaps "Always show" would read better?